pkg/commands/docker.go

```diff
@@ -93,12 +95,34 @@ func (t *dockerBuildTask) Build(runner build.Runner) error {
 		args = append(args, "--build-arg", buildArg)
 	}
 
+	for _, buildSecretArg := range t.buildSecretArgs {
+		args = append(args, "--build-arg", buildSecretArg)
+	}
+
 	if t.contextDir != "" {
 		args = append(args, t.contextDir)
 	} else {
 		args = append(args, ".")
 	}
-	return runner.ExecuteCmd("docker", args)
+
+	return runner.ExecuteCmdWithObfuscation(func(args []string) {
```

Comment:
Can you make this into an actual method - it's an identical concept for compose.

Reply:
Sure. I actually thought about removing all compose codes.